components/job-orchestration/job_orchestration/scheduler/query/query_scheduler.py

```
@@ -87,6 +89,36 @@
+    job_id: str, job_type: int, job_config: dict[str, Any]
+) -> dict[str, Any]:
+    try:
+        query_job_type = QueryJobType(job_type)
+    except ValueError:
+        return {"job_id": job_id, "query_job_type": str(job_type)}
+
+    context: dict[str, Any] = {
+        "job_id": job_id,
+        "query_job_type": query_job_type.to_str(),
+    }
+    if QueryJobType.SEARCH_OR_AGGREGATION == query_job_type:
```

Ruff (0.15.21)

[warning] Yoda condition detected

(SIM300)